src/recommenderapp/app.py

```diff
@@ -17,29 +17,127 @@
 #pylint: disable=wrong-import-position
 from src.prediction_scripts.item_based import recommend_for_new_user
 #pylint: enable=wrong-import-position
-
+import db
 
 app = Flask(__name__)
 app.secret_key = "secret key"
 
 cors = CORS(app, resources={r"/*": {"origins": "*"}})
 
+# Connect to the database
+db.connect()
+# Initialize the database for users
+db.mutation_query('''
```

Comment:
Would it be better to make the code modular and move the queries in a different file? We might face issues with the linter if the code lines go beyond 1000

Reply:
I think we can look into this later. Our app.py is at 200 lines of code right now, so I would be quite surprised if it went above 1000

Reply:
Okay!